libs/ui/map/jest.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   displayName: 'ui-map',
   preset: '../../../jest.preset.js',
+  transformIgnorePatterns: ['node_modules/(?!(ol))'],
```

Comment:
can't we do that for all projects ? at root level ?

Reply:
probably, that would be more future-proof